server/server.js

```diff
@@ -0,0 +1,23 @@
+const express = require('express');
+const path = require('path');
+require('dotenv').config();
```

Comment:
I would consider changing this to `import "dotenv/config"`; using require as little as possible is something my mentor has been encouraging me to do as it's older and can sometimes cause issues.

Reply:
went ahead and fixed it. thanks!